build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    std::process::Command::new("git").arg("submodule").arg("update").arg("--init").arg("--recursive").output().expect("Failed to clone submodules");
```

Comment:
Interesting idea, but I'd prefer to avoid having the direct dependency to whisper.cpp (submodule) since whisper-rs-sys already does the same thing. Once I ship something of value I'd definitely submit fixes to upstream project to avoid having to build cpp libs by myself.